src/error_stats_api_endpoints.py

```diff
@@ -46,6 +47,7 @@ def get_percentile_from_error_stats(
                   'workflow_name': workflow_name,
                   'workflow_ver': workflow_version,
                   'percentile_value': percentile_value,
+                  'standing_laying': standing_laying,
```

Comment:
Check if standing_laying is None, then don't use standing_laying in params.
else use standing_laying in params